src/Shared/classes/RojoResolver.ts

```diff
@@ -173,11 +173,12 @@ export class RojoResolver {
 			let configJson: unknown;
 			try {
 				configJson = JSON.parse(fs.readFileSync(realPath).toString());
-			} catch (e) {}
-			if (isValidRojoConfig(configJson)) {
-				this.parseTree(path.dirname(rojoConfigFilePath), configJson.name, configJson.tree, doNotPush);
-			} else {
-				warn(`RojoResolver: Invalid configuration! ${ajv.errorsText(validateRojo.get().errors)}`);
+			} finally {
```

Comment:
Could you explain this change?

Reply:
This was part due to eslint not allowing an empty catch block. This made me rewrite it to look nicer. This way, it's obvious that the `isValidRojoConfig` check is related to the `configJson` parsing.